packages/shared/src/components/plus/PlusList.tsx

```diff
@@ -1,36 +1,75 @@
 import React, { ReactElement } from 'react';
+import classNames from 'classnames';
 import { ChecklistAIcon } from '../icons';
 import {
   Typography,
   TypographyColor,
+  TypographyProps,
   TypographyTag,
   TypographyType,
 } from '../typography/Typography';
-import { IconSize } from '../Icon';
+import { IconProps, IconSize } from '../Icon';
+import { WithClassNameProps } from '../utilities';
 
-const PlusItem = ({ title }: { title: string }) => {
+interface PlusItemProps {
+  title: string;
+  text?: TypographyProps<TypographyTag.P>;
+  icon?: IconProps;
+}
+
+export const defaultFeatureList = [
+  'Hyper-personalized feed',
+  'AI-powered search',
+  'Squads (developer communities)',
+  'Advanced customizations',
+  '100+ more features',
+];
+
+export const plusFeatureList = [
+  'Ads-free browsing',
+  'Exclusive Plus badge',
+  'Private squad for Plus members',
+  'Support the team and make us smile',
+  'And so much more coming soon...',
+];
+
+const PlusItem = ({ title, text, icon }: PlusItemProps) => {
   return (
     <li className="flex gap-2">
-      <ChecklistAIcon className="text-text-quaternary" size={IconSize.Small} />
+      <ChecklistAIcon
+        className="text-text-quaternary"
```

Comment:
I'm not sure about other places, but here, I think the color should be `tertiary`.

<img width="1299" alt="Screenshot 2024-11-23 at 2 05 43 PM" src="https://github.com/user-attachments/assets/b1fe693c-1941-498b-b5ab-dcbd314d95db">